cleanlab/filter.py

```diff
@@ -406,9 +411,9 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
-def _find_label_issues_multilabel(
-    labels: list,
-    pred_probs: np.ndarray,
+def _keep_at_least_n_per_class(
+    prune_count_matrix: np.ndarray, n: int, *, frac_noise: float = 1.0
+) -> np.ndarray:
     return_indices_ranked_by: Optional[str] = None,
     rank_by_kwargs={},
     filter_by: str = "prune_by_noise_rate",
```

Comment:
Looks like the merge conflicts weren't resolved properly. There's a syntax error on line 426.

```
cleanlab/filter.py:426: error: unmatched ')'  [syntax]
```